www/docs/deploy-app.html

```diff
@@ -0,0 +1,126 @@
+        </li>
+      </ul>
+    </div><!--/.nav-collapse -->
+  </div>
+</nav>
+
+
+<div class="container">
+
+  <div class="row">
+
+    <div class="col-sm-3">
+
+      <ul class="nav nav-bordered nav-stacked">
+        <li ><a href="/docs">Introduction</a></li>
+        <li class="nav-divider"></li>
+        <li class="nav-header">Installation</li>
+        <li ><a href="/docs/install.html">Install Intercity</a></li>
+        <li ><a href="/docs/initial-setup.html">Initial setup</a></li>
+        <li class="nav-divider"></li>
+        <li class="nav-header">Getting started</li>
+        <li ><a href="/docs/install-server.html">Install a server</a></li>
+        <li ><a href="/docs/add-app.html">Add an app</a></li>
+        <li ><a href="/docs/configure-domain-name.html">Configure domain name</a></li>
+        <li ><a href="/docs/add-database.html">Add a database</a></li>
+        <li ><a href="/docs/add-deploy-key.html">Add deploy key</a></li>
+        <li class="active"><a href="/docs/deploy-app.html">Deploy your app</a></li>
+      </ul>
+
+    </div>
+
+    <div class="col-sm-9 docs-content">
+
+      <h1 class="m-t-0">Deploy your app</h1>
+
+<p>Awesome, you’re at the final step. With Intercity, you’ve installed your app server, added your app, configured its domain name, enabled PostgreSQL and added your deploy to key. It’s now time to deploy your app with an easy git push.</p>
+
+<h2 id="adding-the-git-remote">Adding the git remote</h2>
+
+<p>Every app hosted with Intercity has its own unique deploy URL. You can find the deploy URL on the Info tab of an app in your Intercity instance.</p>
+
+<p><img src="/images/docs-app-info-deploy-url@2x.png" class="img-responsive img-thumbnail" /></p>
+
+<p>Add that URL as a remote called “dokku” to the Git repository of your app’s code:</p>
+
+<div class="highlighter-rouge"><pre class="highlight"><code>$ git remote add dokku dokku@95.85.28.175:my-app
+</code></pre>
+</div>
+
+<h2 id="enabling-migrations-on-deploy">Enabling migrations on deploy</h2>
+
+<p>If you’re deploying a Ruby on Rails app, you probably want your database migrations to be run on every deploy. You can do this by creating a file <code class="highlighter-rouge">lib/tasks/run_migrations.rake</code> in your Rails project with the following content:</p>
+
+<div class="highlighter-rouge"><pre class="highlight"><code>Rake::Task['assets:clean'].enhance do
+  Rake::Task['db:migrate'].invoke
+end
+</code></pre>
+</div>
+
+<h2 id="deploying-your-app">Deploying your app</h2>
+
+<p>Let’s deploy the app! Commit all your recent changes that you want to deploy and run:</p>
+
+<div class="highlighter-rouge"><pre class="highlight"><code>$ git push dokku master
+</code></pre>
+</div>
+
+<p>Dokku, the underlying system that Intercity uses to deploy your apps, will now take care of updating your code, running database migrations, do asset compilation and perform a rolling restart of your app.</p>
+
+
+    </div>
+
```

Comment:
Yeah, still need to figure out how to properly deploy a static site to Dokku without having to pre-generate in the repository :)